apps_product_creator/demo/product_attribute.xml

```diff
@@ -0,0 +1,50 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--Copyright (C) 2017-Today: Odoo Community Association (OCA)-->
+<!--License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).-->
+<odoo noupdate="1">
+
+    <record id="attribute_odoo_version" model="product.attribute">
```

Comment:
Following https://github.com/OCA/apps-store/issues/8#issuecomment-377769239 should be data not demo